spec/features/take_whole_pot_calculator_analytics_spec.rb

```diff
@@ -0,0 +1,60 @@
+RSpec.feature 'Take whole pot calculator analytics', type: :feature, js: true do
+  let(:pot) { '100000' }
+  let(:income) { '7000' }
+  let(:pension) { '5000' }
+  let(:pension_invalid) { '5000000' }
+  let(:pension_frequency) { 'annually' }
+  let(:pension_frequency_label) { 'Annual' }
+
+  scenario 'with invalid input' do
+    enter_invalid_data
+    perform_calculation
+
+    expect(data_layer).to include(
+      'pot' => pot,
+      'income' => income,
+      'pension' => pension_invalid,
+      'pension_frequency' => pension_frequency,
+      'valid' => false
+    )
+  end
+
+  scenario 'with valid input' do
+    enter_valid_data
+    perform_calculation
+
+    expect(data_layer).to include(
```

Comment:
The values we send to the data layer need to be suitable for Javascript not the human entered values. I've pushed up a change to the feature to expose the problem - 16d6b6a9f6e91e4eb21766de9a7b33f46eeee8fe